README.md

```diff
@@ -73,7 +73,10 @@ If you want to create and configure `SpotBugsTask` by own, apply the base plugin
 
 ### Apply to Android project
 
-TBU
+Apply this plugin with the `com.android.application` plugin or `com.android.library` plugin to your project,
+then [`SpotBugsTask`](https://spotbugs-gradle-plugin.netlify.com/com/github/spotbugs/snom/spotbugstask) will be generated for each existing variant.
```

Comment:
I'd just like to point out that the tasks are only created for the production code variants on the Android projects. In Java projects, both production and test source sets gets corresponding tasks. It's relatively easy to do the same thing for Android projects.

Reply:
It would be better to do it :+1:, could you suggest the change making it possible? 

I guess we need to use [TestExtension](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.TestExtension.html), [AppExtension#testVariants](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.AppExtension.html#com.android.build.gradle.AppExtension:testVariants) or [AppExtension#unitTestVariants](https://google.github.io/android-gradle-dsl/current/com.android.build.gradle.AppExtension.html#com.android.build.gradle.AppExtension:unitTestVariants) but not so sure.